lms/envs/common.py

```diff
@@ -374,6 +374,10 @@
 
     # Whether or not the dynamic EnrollmentTrackUserPartition should be registered.
     'ENABLE_ENROLLMENT_TRACK_USER_PARTITION': False,
+
+    # Enable Bundled Purchase Functionality
+    # See LEARNER-493
+    'ENABLE_BUNDLED_PURCHASE_FUNCTIONALITY': False,
```

Comment:
`ENABLE_ONE_CLICK_PROGRAM_PURCHASE`. "Bundle" is not a term we use anywhere externally.